src/api/core/ProxyResultBuilder.ts

```diff
@@ -0,0 +1,41 @@
+import { ProxyResult } from "aws-lambda";
+
+export default class ProxyResultBuilder {
+
+    protected response: Partial<ProxyResult> = {};
+
+    constructor(){
+        this.response.headers = {};
+    }
+
+    public status(status: number): ProxyResultBuilder {
+        this.response.statusCode = status;
+        return this;
+    }
+
+    public body(body: string | object): ProxyResultBuilder {
+        if (typeof(body) === 'string'){
+            this.response.body = body;
+        } else if (typeof(body) === 'object') {
+            this.response.body = JSON.stringify(body);
+            this.header('Content-Type', 'application/json')
+        }
+
+        return this;
+    }
+
+    public header(headerKey: string, headerValue: string): ProxyResultBuilder {
+        this.response.headers[headerKey] = headerValue;
```

Comment:
@bigLucas 
All the `this.response.headers` has the same error. I'm still looking, but so far the only way I've found to ignore it was by setting `"strictNullChecks ": false`.

> Error in lines 28, 34 e 35
> `Object is possibly 'undefined'.`

Reply:
@Aristevao you can set `"strictNullChecks ": false`, because we are initializing the object with this code: `this.response.headers = {};` inside the constructor.